src/components/App.tsx

```diff
@@ -134,12 +134,26 @@ const App: FC<AppProps> = ({ demo }) => {
     );
   };
 
+  function getDemoId(): number {
```

Comment:
This works, but I'm wondering if there's an easier/cleaner way to do it in the router in index.js, something like a urlParam: https://reacttraining.com/react-router/web/example/url-params .

Reply:
refactored to use params